firmware/hw_layer/drivers/can/can_msg_tx.cpp

```diff
@@ -87,20 +90,6 @@ void CanTxMessage::setBit(size_t byteIdx, size_t bitIdx) {
 	m_frame.data8[byteIdx] |= 1 << bitIdx;
 }
 
-#else
```

Comment:
why this change?

Reply:
if we can remove code and tests are still passing it must be unneeded code?

Reply:
but the code didn't get removed, it just got #ifdef'd in every single function instead of all at once

Reply:
@mck1117 please forgive me I was excited to try if this would have fixed PCAN. CAN defines are a mess at the moment, this is technical debt right now while I am excited about getting PCAN ready.